cmd/hard/main.go

```diff
@@ -0,0 +1,280 @@
+	rootCmd.AddCommand(npmCmd)
+	rootCmd.AddCommand(yarnCmd)
+}
+
+func init() {
+	homeDir, _ := os.UserHomeDir()
+	hardPath = filepath.Join(homeDir, ".hard")
+
+	// Load .env file
+	if err := loadEnv(filepath.Join(hardPath, ".env")); err != nil {
+		fmt.Println("Error loading .env file:", err)
+		os.Exit(1)
+	}
+
+	wwwPath = os.Getenv("WWW_PATH")
+	if wwwPath == "" {
+		fmt.Println("ERROR: WWW_PATH not set in .env file")
+		os.Exit(1)
+	}
+}
+
+func loadEnv(filename string) error {
+	// For simplicity, we'll just check if the file exists and load it if it does
+	if _, err := os.Stat(filename); os.IsNotExist(err) {
+		return fmt.Errorf("ERROR: Couldn't find '.env' in the hard directory")
+	}
+	err := godotenv.Load(filename)
+	if err != nil {
+		return fmt.Errorf("Error loading .env file: %v", err)
+	}
+
+	return nil
+}
+
+func dockerComposeCmd() *exec.Cmd {
+	dockerComposePath := filepath.Join(hardPath, "docker-compose.yml")
+	if _, err := exec.LookPath("docker"); err != nil {
+		fmt.Println("ERROR: Docker is not installed or not in PATH")
+		os.Exit(1)
+	}
+
+	if _, err := exec.LookPath("docker-compose"); err == nil {
+		return exec.Command("docker-compose", "-f", dockerComposePath)
+	}
+
+	return exec.Command("docker", "compose", "-f", dockerComposePath)
+}
+
+var upCmd = &cobra.Command{
+	Use:   "up",
+	Short: "Start the environment",
+	Run: func(cmd *cobra.Command, args []string) {
+		dcCmd := dockerComposeCmd()
+		dcCmd.Args = append(dcCmd.Args, "up")
+		if detach, _ := cmd.Flags().GetBool("detach"); detach {
+			dcCmd.Args = append(dcCmd.Args, "-d")
+		}
+		dcCmd.Args = append(dcCmd.Args, args...)
+		dcCmd.Stdout = os.Stdout
+		dcCmd.Stderr = os.Stderr
+		dcCmd.Run()
+	},
+}
+
+var downCmd = &cobra.Command{
+	Use:   "down",
+	Short: "Stop the environment",
+	Run: func(cmd *cobra.Command, args []string) {
+		dcCmd := dockerComposeCmd()
+		dcCmd.Args = append(dcCmd.Args, "down")
+		dcCmd.Stdout = os.Stdout
+		dcCmd.Stderr = os.Stderr
+		dcCmd.Run()
+	},
+}
+
+var restartCmd = &cobra.Command{
+	Use:   "restart",
+	Short: "Restart the environment",
+	Run: func(cmd *cobra.Command, args []string) {
+		dcCmd := dockerComposeCmd()
+		dcCmd.Args = append(dcCmd.Args, "restart")
+		dcCmd.Stdout = os.Stdout
+		dcCmd.Stderr = os.Stderr
+		dcCmd.Run()
+	},
+}
+
+var psCmd = &cobra.Command{
+	Use:   "ps",
+	Short: "Display the status of all containers",
+	Run: func(cmd *cobra.Command, args []string) {
+		dcCmd := dockerComposeCmd()
+		dcCmd.Args = append(dcCmd.Args, "ps")
+		dcCmd.Stdout = os.Stdout
+		dcCmd.Stderr = os.Stderr
+		dcCmd.Run()
+	},
+}
+
+var bashCmd = &cobra.Command{
+	Use:   "bash",
+	Short: "Start a shell session within the app container",
+	Run: func(cmd *cobra.Command, args []string) {
+		dcCmd := dockerComposeCmd()
+		dcCmd.Args = append(dcCmd.Args, "exec", "app", "bash")
+		dcCmd.Stdout = os.Stdout
+		dcCmd.Stderr = os.Stderr
+		dcCmd.Stdin = os.Stdin
+		dcCmd.Run()
+	},
+}
+
+var laravelCmd = &cobra.Command{
```

Comment:
Not working properly. Hard cli don't what to do with "-" flag 

![image](https://github.com/user-attachments/assets/ee2e6074-d4d4-4f3c-add6-8d5b16b61f12)

Reply:
Now all command flags are passed to the command within docker